hi-ml-histopathology/testhisto/testhisto/utils/test_plots_utils.py

```diff
@@ -58,9 +58,7 @@ def assert_plot_func_called_if_among_plot_options(
         },
     ],
 )
-def test_plots_handler_plots_only_desired_plot_options(
-    plot_options: Collection[PlotOption],
-) -> None:
+def test_plots_handler_plots_only_desired_plot_options(plot_options: Collection[PlotOption],) -> None:
```

Comment:
following up on @ant0nsc 's comment: this test was not failing also before because it operates on a single GPU while the issue is with DDP, right? do we have a way to test outputs in DDP?

Reply:
The test didn't fail because I am mocking "save_confusion_matrix". Also, this test makes sure that we're calling the right functions for a set of plot_options. The plot functions are tested separately in test_viz_utils.py.

The outputsHandler deals with DDP synchronization for which we already have tests. The plots handler is meant to be executed from rank=0 only, not sure if we should test it in DDP because we'll have to rewrite the logic from outputs handler in the test....